### DISCUSSION TITLE
subscriptionPricePoints endpoint updates

### DISCUSSION THREAD

**Discussion by IgorNovikov10** [hyochan/react-native-iap#2900] [Q&A]
Few months ago we received such a message from Apple

`Hello, We're reaching out to let you know that the subscriptionPricePoints endpoint now requires a territory relationship filter. If you use the App Store Connect API to configure your subscriptions, please note that you'll be required to define this filter in an upcoming release. View App Store Connect API release notes. If you have any questions, please contact us. The Apple Developer Relations Team`

We use the following methods `purchaseErrorListener, purchaseUpdatedListener, finishTransaction, initConnection, getSubscriptions` provided by react-native-iap package to get available subscriptions from the App store. Do we need to make any additional actions due to the information in the message?

**Comment by axmo** [comment]
This sounds like it might be the cause of: https://github.com/hyochan/react-native-iap/issues/2914 ?

or maybe not? A quick search of the repo turned up no code with `subscriptionPricePoints`
